ampd/src/queue/queued_broadcaster.rs

```diff
@@ -108,35 +91,40 @@ where
 
         loop {
             select! {
-              msg = rx.recv() => match msg {
-                None => break,
-                Some(msg) => {
-                  let fee = broadcaster.estimate_fee(vec![msg.clone()]).await.change_context(Error::EstimateFee)?;
```

Comment:
So it looks like before, this loop would exit if there was a fee estimation error. Whereas now, the loop continues. Why the change?

Reply:
It ignores tx that cannot be estimated and sends the error back to the client.